src/program.cpp

```diff
@@ -741,7 +390,7 @@ void program::perf_report(std::ostream& os, std::size_t n, parameter_map params)
 void program::debug_print() const { std::cout << *this << std::endl; }
 void program::debug_print(instruction_ref ins) const
```

Comment:
I think the `debug_print` with an instruction should be done in the module class. Do we use this function from the program?

Reply:
We can probably leave it for now.